### DISCUSSION TITLE
버그 및 개선 사항 취합하기

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/prolog#782] [General]
## 코멘트로 남길 내용
- 메타 인지 향상을 위해 제도적 개선이 필요한 부분
- 프롤로그 서비스 사용성 향상을 위해 기술적 개선이 필요한 부분
- 프롤로그 서비스를 사용하며 발견한 버그
- UI / UX를 개선하고 싶은 부분

## 코멘트 작성 방법
- 버그 내용은 코멘트 상단에 [버그]를 붙인다.
- 개선사항은 코멘트 상단에 [개선]을 붙인다.

**Comment by JUDONGHYEOK** [comment]
## [개선] 
- 오류메세지를 alert대신 모달창으로 대체하면 좋을 것 같습니다. 
<img width="1452" alt="스크린샷 2022-04-22 오후 6 11 10" src="https://user-images.githubusercontent.com/65863017/164675438-35e4d011-6716-4ec2-9a0b-fa149c4d178a.png">

- sorting 기능이 추가되었으면 좋을 것 같습니다.

- 내 학습로그를 볼 때 미리보기 에서 마크다운 문법은 안보이는 게 좋을 것 같습니다!
<img width="1188" alt="스크린샷 2022-04-22 오후 6 19 00" src="https://user-images.githubusercontent.com/65863017/164676716-77a15497-d55f-4a36-ac2f-40c7a462cd72.png">

## [버그]
- 없는 URL 관련하여 기본 템플릿이 보이는 것 같습니다. 404 페이지가 추가되면 좋을 것 같습니다.
<img width="1728" alt="스크린샷 2022-04-22 오후 6 10 31" src="https://user-images.githubusercontent.com/65863017/164675344-e4bcb6f7-904b-4c6e-a89f-6c04057ffbc4.png">

- 다른 사람의 프로필을 들어간 후 제 프로필로 접속하면 태그와 자기소개가 다른사람 프로필이 유지가 됩니다
<img width="1728" alt="스크린샷 2022-04-22 오후 6 09 36" src="https://user-images.githubusercontent.com/65863017/164675203-d5b7c2d8-6c82-4687-85bc-db783db10837.png">

**Comment by wonsss** [comment]
## [버그]
- 모바일 반응형 보기 시, 긴 타이틀이 영역 밖으로 벗어납니다.
<img width="481" alt="스크린샷 2022-04-22 오후 6 31 18" src="https://user-images.githubusercontent.com/59413128/164678841-8c31c75d-bf2b-4846-9b0d-4298dd244097.png">

## [개선]
- '메인페이지-인기있는학습로그' 영역에서 해시태그 부분에 불필요한 scrollbar가 표시되는데, 개선할 필요가 있을 것 같아요. 
<img width="1096" alt="스크린샷 2022-04-22 오후 6 08 59" src="https://user-images.githubusercontent.com/59413128/164675082-001f0d3a-6c41-495b-907c-8989b8708a4f.png">

- 글 보기 시, 하단의 해시태그들이 다 붙어있어서 가독성이 떨어지는 것 같아요. 
<img width="459" alt="스크린샷 2022-04-22 오후 6 09 55" src="https://user-images.githubusercontent.com/59413128/164675253-ab7751ad-5c09-48af-882f-b6e0d4b97add.png">

- 글 보기 시, 하단의 해시태그를 누르면 해당 해시태그가 있는 게시물들을 모아서 보여주면 좋을 것 같아요.
- 글 수정 시, 수정 영역이 너무 작은 것 같아요. 수정 영역을 viewport 전체 높이로 확대할 것을 제안합니다
- 메인화면 카드 슬라이더의 오른쪽에 다음 카드가 살짝 보이는데, 의도한 걸까요? 의도하지 않았다면 현재 보는 카드의 너비를 전체 영역으로 설정하는 것이 더 깔끔해보일 수도 있을 것 같아요.
<img width="1433" alt="스크린샷 2022-04-22 오후 6 12 04" src="https://user-images.githubusercontent.com/59413128/164675580-b82ef164-8129-4e88-ade1-12562dfa0a44.png">

- 카테고리 검색 시, 검색어 입력 후 엔터를 누르면 새로고침이 되는데, 새로고침은 불필요할 것 같아요.
<img width="552" alt="스크린샷 2022-04-22 오후 6 21 59" src="https://user-images.githubusercontent.com/59413128/164677245-277a6841-3f24-434e-b34d-1581728e80a8.png">

- 역량 페이지에서, `추가`, `수정`, `삭제` 버튼이 한 줄에 오도록 감싸는 컨테이너 너비 조정
<img width="1099" alt="스크린샷 2022-04-24 오후 1 48 50" src="https://user-images.githubusercontent.com/59413128/164957141-67930e1f-c8b7-4604-a82c-29be0c308d7e.png">

- 리포트 페이지에서, 가중치 인풋에 숫자를 입력하면, 해당 폼의 하단에 현재 입력된 가중치들의 합이 얼마이며 되어야 하는 가중치 합(20)에 대해서도 표시해주는 것이 사용자 경험에 좋을 것 같아요.
<img width="927" alt="스크린샷 2022-04-24 오후 2 06 32" src="https://user-images.githubusercontent.com/59413128/164957527-13d98050-75ed-45b5-b720-2e4627da7c15.png">

- 게시글 작성 시, 이미지 추가 기능 추가
- 게시글 조회수 중복 방지 기능 추가
- 게시글 하단에 관심있을 만한 포스트 추천
- 게시물 공유하기 기능 버튼 추가
- 댓글 기능 추가
- 크루 구독 및 구독한 글 피드 보기 페이지 추가
- `알림` 기능 추가하기
	-  본인 게시글에 `좋아요`가 눌리면, 썸네일 옆에 추가한 종 모양의 알림 버튼에 아직 확인하지 않은 `좋아요` 알림 숫자가 표시된다. 해당 `알림 버튼`을 클릭하면 `좋아요`를 포함하여 여러 알림(향후 추가될 수 있는)이 쌓여있는 페이지를 표시한다.
- 검색 폼을 헤더로 옮겨서 어느 페이지에서던 표시되도록 고정하기
	- 현재는 `홈` 페이지나 `게시글`에 있을 때, 검색하려면  다시 `학습로그`페이지에 들어가야 하는 번거로움이 있어요.
<img width="484" alt="스크린샷 2022-04-24 오후 1 52 35" src="https://user-images.githubusercontent.com/59413128/164957266-21271ab6-b929-40bc-a89c-0fdb2e5b8fc4.png">

- 그리고 헤더에서 `홈` 버튼은 왼쪽에 로고 `PROLOG`와 동일한 기능을 하므로, `홈`버튼은 빼도 무방할 것 같아요.
- 리포트 타임라인에서, 
	- 등록된 리포트 박스에 각각 학습역량 가중치 점수들을 표시하기
	- 누적 학습역량 가중치 점수를 타임라인의 최상단에 고정하여 표시하기
- 새 리포트 작성 시, 기간 선택에 대하여 몇 가지 제약을 추가하기
	- 일주일(월~일) 단위로만 선택 가능하도록 수정(기간 선택 캘린더를 누르면 월요일부터 일요일까지의 일주일 단위로만 영역 지정되어 사용자가 이 규칙 안에서만 선택하도록 도운다)
		- 이유 : 리포트 등록 기간이 일정해야 리포트별 학습역량 가중치 점수가 의미를 가질 수 있을 것 같아요.
- 헤더에 `리포트` 버튼 추가하여, 모든 사용자들의 리포트들을 최신순으로 볼 수 있는 페이지로 이동할 수 있게 한다.

**Comment by kwannee** [comment]
### [개선] 
1. UI 메인 화면 `인기있는 학습로그` 카드 hover 시 미션의 폰트 컬러가 변경되는데, 오히려 가시성이 떨어집니다.
![image](https://user-images.githubusercontent.com/41886825/164658903-382c3b16-7b5f-4ae0-8531-5cecdb58b857.png)

2. UI  학습로그 게시판 제일 밑 pagination의 밑 부분이 너무 딱 붙어있습니다. 클릭하는데 어려움이 있을 것 같습니다.
![image](https://user-images.githubusercontent.com/41886825/164672840-a7254536-c40a-4c55-abbb-a99bb20817c7.png)

3. UI `내 프로필`에서 페이지 변경 시 header width가 페이지에 따라 달라져서 사용자 입장에서 계속 위치가 바뀝니다. 큰 건 아니지만 사소한 디테일도 챙기면 좋은 서비스가 될 것 같습니다.
![image](https://user-images.githubusercontent.com/41886825/164675853-9c9eaaf0-e720-4881-aa8a-414f30273497.png)
![image](https://user-images.githubusercontent.com/41886825/164676611-14b858bb-c99c-4aaf-a723-55a48e5b09f8.png)

**Comment by wishoon** [comment]
### [개선]

1. 검색 시스템의 사용성 개선
- 예를 들어, `JWT`를 입력했을 경우 관련이 없는 `Jenkins`에 대한 내용도 같이 검색이 됩니다.
    - 해당 학습로그에 작성되어 있는 `gradle` 설정의 `jwt` 때문에 검색이 되는 것으로 보여지기 때문에 개선이 필요할 것으로 보입니다.
  
<img width="1057" alt="image" src="https://user-images.githubusercontent.com/48710213/164740177-246a01c3-c185-45ee-a6c6-3d3bce897e1b.png">

2. 단어 검색시 작성된 단어의 일부분을 기반으로 유사단어를 보여주는 기능
- 예를 들어, `객체`라는 키워드 입력 시 `객체지향`, `객체지향 시스템` 등을 보여주도록 기능 개선을 제안해봅니다.

- 현재 모습
<img width="1052" alt="image" src="https://user-images.githubusercontent.com/48710213/164740042-452d31c3-b486-4c9d-a13a-43d045cd68c4.png">

- 개선 방향 제안
<img width="1056" alt="image" src="https://user-images.githubusercontent.com/48710213/164742016-84857b2d-76c7-4979-b676-4116b28a376f.png">


3. 필터 검색
- 필터 검색을 위해 키워드 입력 후, 검색 결과를 클릭 하지 않고 키보드를 통해 엔터를 입력하면 페이지가 렌더링되는 현상
    - 클릭을 해야 검색이 되지만, 키보드를 통해 엔터를 입력하면 페이지가 새로고침 되기 때문에 사용자에게 검색이 되었다는 착각을 줄 수 있어 개선이 필요할 것으로 보입니다.
 
<img width="1060" alt="image" src="https://user-images.githubusercontent.com/48710213/164739646-3032caa7-4401-4fdf-84e2-8798f0060b9c.png">

<img width="1059" alt="image" src="https://user-images.githubusercontent.com/48710213/164739749-bea41c8a-72ff-4539-9014-94b68cc26a6f.png">

4. 작성된 글을 계속 클릭하면 계속 방문집계가 되는 문제
- 이미 방문하였으면 일정 기간 동안 카운팅이 안되도록, 혹은 방문시 카운팅이 안되도록 개선을 제안해봅니다.

<img width="1059" alt="image" src="https://user-images.githubusercontent.com/48710213/164741608-e230d29a-cdcd-47ed-a148-3fdc1cc9255e.png">

<img width="1053" alt="image" src="https://user-images.githubusercontent.com/48710213/164741703-272f0a30-47c8-43eb-8093-a3a4d1b083f0.png">

### [버그]

1. 트랙패드 뒤로가기 모션
- [글 목록 페이지](https://prolog.techcourse.co.kr/studylogs), [본인이 작성한 학습로그 페이지] 에서 뒤로가기 모션을 3번 수행해야 이전 페이지로 이동하는 문제가 있습니다. (사용기기 : m1 에어, 크롬)

**Comment by her0807** [comment]
### [추가]
1. 리포트 작성 방법 설명문 추가
- 처음 리포트를 작성하러 들어가서 느낀 점은 `어떻게 하는거지?` 였습니다. 
 리포트 작성 시, 예시와 함께 `역량 그래프의 합은 20으로 맞춰야 한다` 등, 설명이 추가되었으면 좋겠습니다.
 <img width="1251" alt="스크린샷 2022-04-24 오후 12 22 26" src="https://user-images.githubusercontent.com/26570275/164955127-68f8ab99-7ced-48ed-9c2f-e4a9296b673a.png">


### [에러]
2. 프롤로그 역량 - 학습로그에 역량 매핑하기 
- `학습로그에 + 버튼을 누르고` ,  `역량을 선택`한 후`역량 등록`을 누른 뒤 `다시 + 버튼`을 누르면
이전에 등록한 역량이 선택되어 있었으면 좋겠습니다. 현재는 초기화 되는 것 같습니다!

<img width="1194" alt="스크린샷 2022-04-24 오후 12 32 51" src="https://user-images.githubusercontent.com/26570275/164955349-28439a33-0330-4547-be4e-66618cb4e6a0.png">

### [개선]
3. 전체 검색 시  `levels, missions, tags, members ` 의 모든 값을 포함한 결과 노출

- 제일 상단의 검색 기능을 사용할 때, 현재는 내용을 바탕으로 검색이 되는 것 같습니다. 
levels, missions, tags, members 선택 없이 전체를 포함한 결과가 나오게 했으면 좋겠습니다.
ex) 수달 검색 시, 수달이 작성한 글, 태그에 수달이 포함된 걸, 내용에 수달이 포함된 글이 검색된다. 


<img width="1194" alt="스크린샷 2022-04-24 오후 12 41 15" src="https://user-images.githubusercontent.com/26570275/164955525-387d9cf4-0ea5-419b-aa48-91f246a67c3c.png">

**Comment by leo0842** [comment]
[개선] 

1. 리포트 수정에서 가중치 수정이 안됩니당!
<img width="835" alt="image" src="https://user-images.githubusercontent.com/42317507/164964583-b678c1a3-9137-459c-82f8-66c7192965cf.png">


2. 레포트 타임라인이 현재는 레포트 생성순으로 보이는데 이게 레포트 생성순으로 되어야 할지, 레포트 기간순으로 되어야 할지 이야기해보고 싶습니다.
<img width="823" alt="image" src="https://user-images.githubusercontent.com/42317507/164964826-638884c4-aa84-4a2d-8c35-01ee9cef3d4e.png">

**Comment by EastHShin** [comment]
### [개선]
1.
![image](https://user-images.githubusercontent.com/64204666/164965086-fc8967a3-b93e-4a54-87d5-23cc4009edd3.png)
글쓰기를 할 때 카테고리 선택시 스크롤을 내려가며 일일이 찾아 주어야 하는데, 분야/레벨/미션 별로 카테고리를 세분화하여 선택할 수 있으면 좋겠습니다!

2.
![image](https://user-images.githubusercontent.com/64204666/164965715-48c8e933-0db6-4a10-9a58-334eb2524479.png)
글 목록에서 멤버 별 선택 시, 계정이 생성된 순서로 닉네임이 배치된 것으로 보이는데요! 가나다 순으로 바꾸면 어떨까 의논해보면 좋겠습니다!

**Comment by al-bur** [comment]
테스트 환경
- window

## [개선]

**1. 불필요해보이는 스크롤바 제거**

![카드](https://user-images.githubusercontent.com/64825713/164965373-d062b04c-690c-4789-9072-7d9904ac54a0.png)

- 이미지의 노란색 하이라이팅 부분은 불필요해보입니다.
- 모든 곳에서 해당 스크롤바가 사용되고 있지 않으니 화면에 렌더링이 되지 않도록 하는 게 좋을 것 같습니다.


**2. 작성중인 글 저장하는 기능**

- 현재는 글을 작성할 때 중간에 저장이 불가능합니다.
- 대부분, 한 번에 글을 작성하는 것이 아니기 때문에 중간에 저장할 수 있는 `작성중인 글 저장 기능`이 있으면 좋겠습니다.

**Comment by euijinkk** [comment]
불편한점, UI/UX 적으로 아쉬운점 위주로 적어보았습니다! 

## [개선] 
1. 바닥 마진 추가
바닥에 마진이 없어서 UI가 이쁘지 않습니다.
<img width="914" alt="image" src="https://user-images.githubusercontent.com/24906022/164974990-7a7992f0-abef-4782-8031-c90c7e10c6ff.png">

2. 배너 반응형
배너 반응형이 구현되어 있지 않습니다.
<img width="747" alt="image" src="https://user-images.githubusercontent.com/24906022/164975072-ed074d70-5408-4dad-9c74-1fe5a755b63d.png">

4. 게시물 전체 font 크기 및 스타일 조정
한 페이지에 3개의 게시물만이 떠있는데, 보면 padding 및 margin이 너무 커 보입니다.
<img width="1206" alt="image" src="https://user-images.githubusercontent.com/24906022/164975107-08325ac4-1b34-4265-88f2-133cc20bf539.png">

5. 더보기 버튼이 위에 있어서, 바닥까지 스크롤 했다가, 더보기 버튼을 찾으러 다시 올라가야하는 UX 문제
<img width="1148" alt="image" src="https://user-images.githubusercontent.com/24906022/164975166-0ddb1324-6539-40be-827f-3f5c761db3db.png">

6. filter 검색시에, input에 자동 focus 기능
<img width="379" alt="image" src="https://user-images.githubusercontent.com/24906022/164975182-47045db9-2ffb-4955-bd7c-b28e0a843560.png">

7. filter 검색시에 enter 먹지 않음.

8. Not Found UI 개선
UI가 너무 단순해서 아쉬워요! 시간이 나면 개선하고 싶어요~
<img width="1141" alt="image" src="https://user-images.githubusercontent.com/24906022/164975239-8690f5e4-a07f-4e9b-8e9a-a989ce0af6b2.png">

9. 글 작성 시, 미션 선택에서는 KeyBoard Event (ArrowUp, ArrowDown)가 동작하지 않음.

10. 미션 선택에서 학습로그만 존재함. 학습로그와 독립적인 내용 (회고, 철학 등)에 관한 내용은 선택하지 못 함. 하지만 간혹 크루들이 회고나 개발자 철학에 대해서도 글을 써주곤 함. 카테고리를 아무거나 선택하고 글을 쓰는 듯 함.


12. 역량 탭 반응형 구현
임의의 p태그로 구현되어있음.
<img width="655" alt="image" src="https://user-images.githubusercontent.com/24906022/164975850-368bcd46-7a1d-43c3-8b69-ca627d706ba8.png">

13. 스크랩 글로 가는 과정이 복잡하다고 느껴짐.
스크랩 탭은 자주 이용되면 좋겠다고 생각이 드는데,
내 프로필에 들어가서 스크랩 을 누르는 2번의 과정이 필요함. 더욱이, 맨 아래 구석에 숨겨져 있어 잘 이용하지 않게 됨.

## [버그]
1. 태그 선택 시 ArrowDown 처음 누르면, 2칸이 이동하거나, 1칸도 이동하지 않는 현상
![화면 기록 2022-04-24 오후 9 01 14](https://user-images.githubusercontent.com/24906022/164975510-3e1becd5-4185-4fe0-b364-5fb40749f71b.gif)

2. 배너가 carousel indicator를 침범함.
<img width="920" alt="image" src="https://user-images.githubusercontent.com/24906022/164975551-b72d0000-db69-4e8d-bd11-4f7061dc396a.png">

3. 역량탭 클릭시 ScrollToTop 되지 않고, 스크롤 위치 유지됨.
![화면 기록 2022-04-24 오후 9 09 14](https://user-images.githubusercontent.com/24906022/164975814-2d64e48f-64f2-458f-a772-161f24e66104.gif)

4. 내 프로필 - 학습로그 탭에서 뒤로가기가 동작하지 않음
3~4번 뒤로가기 눌러야 뒤로가기가 되네요
![화면 기록 2022-04-24 오후 9 17 37](https://user-images.githubusercontent.com/24906022/164976163-d5a04e54-9a05-4564-988d-264ff11ca339.gif)

5. 프로필에 들어갈 때 ScrollToTop 되지 않음
![화면 기록 2022-04-24 오후 9 19 33](https://user-images.githubusercontent.com/24906022/164976185-3eb9104f-9b04-4983-985a-108f7373abf2.gif)

**Comment by verus-j** [comment]
### [개선] 학습 로그 페이지에서 새로고침 시 조회수 증가 수정
![스크린샷 2022-04-24 오후 11 37 06](https://user-images.githubusercontent.com/25394140/164981719-611fb44f-cb51-4dc8-9e99-020d97455b27.png)
* 조회수를 통해 인기있는 학습로그를 판별하는데, 새로고침을 통한 무차별적인 조회수 증가로 인해 의미 있는 인기 학습 로그를 구분할 수 없음
* 시간이나 계정별로 제한을 두어 조회수를 증가하도록 개선하면 의미 있는 인기 학습 로그를 구분할 수 있을 것으로 생각됨

### [개선] 학습 로그 임시 저장 기능
* 작성 중인 학습 로그를 임시적으로 저장할 수 있도록 기능 추가

### [개선] 학습 로그 검색 정렬 기능 추가
![스크린샷 2022-04-24 오후 11 42 08](https://user-images.githubusercontent.com/25394140/164981922-0f7ca470-7077-440c-bfb5-60c5446440ab.png)
* 학습 로그를 검색하는데 조회수나 좋아요수를 기준으로 정렬할 수 있는 기능 제공

### [개선] 전체적으로 웹 페이지 하단에 빈 공간 추가
![스크린샷 2022-04-24 오후 11 45 23](https://user-images.githubusercontent.com/25394140/164982030-d5c4fcd7-5695-4328-a92a-99b0f8eaa283.png)
* 웹 페이지 하단에 빈 공간이 전혀 없어서, 개인적으로 약간 답답한 느낌이 있음

**Comment by Yboyu0u** [comment]
[개선]
1. 글 임시저장해주는 기능
velog를 사용중인데 편한 기능중 하나가 임시저장을 해주는 거더라고요. 그래서 저희 학습로그도 글을 임시저장해주는 기능이 존재하면 좋을 것 같습니다.

2. 리포트 수정
현재 리포트 수정할 때 날짜와 가중치를 수정불가 처리해 놓으셨는데 이 이유가 궁금합니다. 

3. 검색기능 
학습로그 전체 검색할 때 페어의 이름으로 검색하면 검색이 안되더라고요. 예전부터 조금 불편하다고 생각했는데 한번 의논해 보고 싶습니다. 
(+어떤 기준으로 검색이 되고 있는 건지도 궁금합니다.)

4. 글 정렬
현재 글들이 최신순으로 정렬되어 보여지고 있는 것 같은데 인기순이나 오랜된 순 같이 유저가 선택해서 보여주는 것은 어떨지 의논해 보고 싶습니다. 

5. 리포트 가중치 
![image](https://user-images.githubusercontent.com/63579113/164986277-5b08f5ec-cf59-412c-b40d-f6040095dcb8.png)
가중치가 20이 되어야 한다는 설명이 없어서 처음에 저런 알람이 떠서 헷갈렸는데 따로 설명을 해주면 좋을 것 같습니다.

**Comment by 2yujeong** [comment]
## [개선]
새로운 태그를 등록할 때 해시태그를 먼저 입력하고 단어를 입력하면 단어 앞에 해시태그가 두 개가 되는 현상이 발생합니다.
<img width="274" alt="image" src="https://user-images.githubusercontent.com/76840965/164985139-0e59f1bb-21e3-413a-9fb4-e57319494c91.png">
<img width="228" alt="image" src="https://user-images.githubusercontent.com/76840965/164985083-98074ab8-dc16-4211-a533-7fb364fded0d.png">
새로운 태그를 등록할 때는 자동적으로 단어 앞에 하나의 '#'가 추가로 붙어서 이러한 현상이 발생하는 거 같습니다.

이 경우 태그로 등록할 단어의 앞에 '#'이 추가로 붙은 채로 태그 등록이 되어서 내 프로필에서 태그를 확인할 때도 앞에 해시태그가 두 개 붙은 상태로 나타납니다. 
<img width="317" alt="image" src="https://user-images.githubusercontent.com/76840965/164985231-6029f2fe-67b2-4f94-8a8e-ea2c9bcc1f9c.png">

또한 'tags'를 이용한 검색 시에도 어떤 단어는 앞에 해시태그가 붙어있고 어떤 단어는 붙어있지 않아서 그 기준이 모호하거나 통일성이 떨어진다고 느껴집니다.
<img width="271" alt="image" src="https://user-images.githubusercontent.com/76840965/164985298-775f2ea8-1405-47e5-b193-9e8d06bbe7a8.png">

새로운 태그를 생성할 때 해시태그 없이 단어만 작성하면 문제없지만, 보통 태그를 달 때 앞에 해시태그를 먼저 작성해준 뒤 단어를 작성해주기 때문에 혼란이 발생하기 쉬우므로 개선 요청합니다.

**Comment by hanull** [comment]
### [개선]

1. 검색

- 제목, 내용, 크루명으로 나누어 각각 검색할 수 있으면 좋을 것 같아요.

<img width="847" alt="image" src="https://user-images.githubusercontent.com/46413629/164991617-f26f36fb-59d7-47dc-8767-132adfe84d9f.png">


2. 역량탭

- 탭이 길어져서 스크롤바가 생길 때, 버튼 위치가 변경 됩니다. 수정이 필요할 것 같아요.

<img width="845" alt="image" src="https://user-images.githubusercontent.com/46413629/164991633-a1ac2226-a300-42eb-9957-cb84593880f8.png">

<img width="845" alt="image" src="https://user-images.githubusercontent.com/46413629/164991644-d7c80ea7-6fd9-4049-8240-d34c4470a592.png">


3. 필터 (members)

- 현재는 모든 크루가 한 번에 보여서 특정 크루들만 선택하기 어렵습니다.

- 3기와 4기, 백엔드와 프론트로 나누어서 선택할 수 있도록 변경하면 좋을 것 같아요.
